src/k2/json/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+file(GLOB HEADERS "*.h")
+
+add_library(json INTERFACE)
```

Comment:
Do you still need to add json.hpp to the repo? It looks like you aren't using the macro the new version provides unless I missed it, so we can stick to the installed Ubuntu version.

Reply:
done